components/detail_view.py

```diff
@@ -0,0 +1,65 @@
+import dash
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output
+from data_handling import LoadedData
+from resources.constants import Constants
+import numpy as np
+
+
+class DetailView:
+    current_notes = ""
+
+    @staticmethod
+    def get_component():
+        widget = dbc.Col(
+            className="col-3 ml-3 mr-3",
+            children=[
+                dbc.Row(
+                    style={"height": "90%"},  # why doesnt row classes work?????????
```

Comment:
```suggestion
                    style={"height": "90%"},  # why doesnt row classes work?????????
```
Not sure what is meant here?

What options have been tested?